### ISSUE TITLE
Why is the I3D-only ablation result so good — significantly higher than other papers?

### ISSUE THREAD

**Issue by cyuyande** [abrarmajeedi/rica2_aqa#5]
Hello, thank you for sharing your excellent work and code! I downloaded the repository and carefully followed the instructions to reproduce the experiments on the FineDiving dataset.

I conducted an ablation experiment where only the I3D branch was retained while keeping all other settings unchanged. Interestingly, the performance was significantly higher than the other ablation settings and even exceeded some results reported in the paper. Moreover, the obtained I3D-only results are substantially higher than those reported in other AQA studies using I3D as the backbone.

However, I noticed that the paper does not report the results for this specific “I3D-only” setting. Could you please clarify whether this experiment was conducted during your study, and if so, could you share the corresponding results or insights?

Thank you very much for your time and kind help!

<img width="551" height="279" alt="Image" src="https://github.com/user-attachments/assets/e22028a0-b9aa-41da-ba9a-5f93e45e525d" />

**Comment by abrarmajeedi** [comment]
Thanks for your interest in our work! We didn’t run the “I3D-only” setting, so I’m not sure what exact configuration you used. Could you clarify what you mean by “retaining only the I3D branch while keeping other settings unchanged”? For example, did you remove model components like the DAG module, ranking/sparsity losses, or modify training details such as learning rate or batch size?

Also note that AQA performance is evaluated using both SRCC and RL2 — improving one can sometimes reduce the other. For reference, your RL2 (0.3153) exceeds the current state of the art (RL2 = 0.229, Human-Centric Fine-Grained Action Quality Assessment, J. Xu et al., TPAMI). It might be useful to compare your I3D-only setup with baselines from works like TPT, CoRe, or especially the FineDiving repository itself as a sanity check.

**Comment by cyuyande** [comment]
Thank you very much for your kind reply!

To clarify, in my “I3D-only” experiment, I removed the DAG module, the ranking/sparsity losses, and the cross-attention component, while keeping all other settings (learning rate, batch size, optimizer, etc.) unchanged. I only retained the MSE loss, so the model is essentially an I3D backbone directly connected to the regression head.

Thanks again for your time and valuable feedback!   

<img width="630" height="820" alt="Image" src="https://github.com/user-attachments/assets/f0b8b809-103f-4859-9a18-280bcf4a5467" />

<img width="805" height="175" alt="Image" src="https://github.com/user-attachments/assets/9a8ba6d2-6a89-4f0d-80d9-650e707639f8" />

**Comment by abrarmajeedi** [comment]
Okay. I would highly recommend trying the same I3D-only setup with code from baselines from works like TPT, CoRe, or especially the FineDiving repository.
